clio-integration-test/src/main/python/clio_test.py

```diff
@@ -146,23 +147,43 @@ def testWithout(header, status):
 
 def test_read_group_metadata():
     id = str(uuid.uuid4()).replace('-', '')
-    library = 'library' + id
+    expected = {
+        'flowcell_barcode': 'barcode1',
+        'lane': 1,
+        'library_name': 'library' + id,
+        'project': 'testProject'
+    }
+    upsert = {'project' : expected['project']}
+    suffix = '/readgroup/metadata/v1/barcode1/1/' + expected['library_name']
+    upsertResponse = requests.post(clio_http_uri + suffix, json=upsert)
```

Comment:
Until this python is retired and scalafmt reigns supreme, I use IntelliJ's auto-formatter to decide for me: https://www.jetbrains.com/help/idea/reformatting-source-code.html